test-snapshots/packages.dhall

```diff
@@ -6,10 +6,25 @@ in  upstream
   with prelude =
     { dependencies = [] : List Text
     , repo = "https://github.com/purescm/purescript-prelude.git"
-    , version = "63ff5a24beba191c8fda919a58489f36bde2d506"
+    , version = "v6.0.1-scm"
     }
   with partial =
     { dependencies = [] : List Text
     , repo = "https://github.com/purescm/purescript-partial.git"
-    , version = "d8b6daf068e4aab0ad85a42d0f890ed37f416923"
+    , version = "v4.0.0-scm"
+    }
+  with effect =
+    { dependencies = [ "prelude" ]
+    , repo = "https://github.com/purescm/purescript-effect.git"
+    , version = "v4.0.0-scm"
```

Comment:
I'd advise against pointing spago to branches, because new commits won't be picked up over time. Commit hashes are the most reliable.